apis/pkg/meta/v1alpha1/zz_generated.configuration_types.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2020 The Crossplane Authors.
```

Comment:
```suggestion
Copyright 2022 The Crossplane Authors.
```

Reply:
I'm going to leave these as-is.

While the new files were obviously added this year, they're "generated" copies of files added in 2020. To have them include the correct year in the copyright we'd either have to teach the (very basic) shell script that makes the copies to alter the dates, or update the copyright on the original files.